### PR TITLE
Ldap configuration

### PR DESCRIPTION
**Pull request by MatthieuMan**
Ldap configuration test

### CODE REVIEW

File: features/bootstrap/LdapConfigurationContext.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        
+    }
+
+    /**
+     * @Then this configuration has disappeared from the LDAP configuration list
+     */
+    public function thisConfigurationHasDisappearedFromTheLdapConfigurationList()
+    {
+        $this->page = new LdapConfigurationListingPage($this);
+
```
Comment (kduret) [inline features/bootstrap/LdapConfigurationContext.php]: You need to check that configuration has been deleted

File: features/bootstrap/LdapConfigurationContext.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        
+    }
+
+    /**
+     * @Then all changes are saved
+     */
+    public function allChangesAreSaved()
+    {
+        $this->page->save();
+
```
Comment (kduret) [inline features/bootstrap/LdapConfigurationContext.php]: You need to check that properties have been correctly saved

File: features/bootstrap/LdapConfigurationContext.php (hunk 1 above)
Comment (MatthieuMan) [inline features/bootstrap/LdapConfigurationContext.php]: yes sorry

File: features/bootstrap/LdapConfigurationContext.php (hunk 2 above)
Comment (MatthieuMan) [inline features/bootstrap/LdapConfigurationContext.php]: yes sorry

**Pull request merged**